lib/phlex/sgml.rb

```diff
@@ -72,6 +72,10 @@ def __final_call__(buffer = nil, target: +"", view_context: nil, parent: nil, &b
 			buffer ? (buffer << target) : target
 		end
 
+		def process_attributes?
```

Comment:
this should be private

Reply:
Is there a need for the additional `process_attributes?` method, as opposed to simply using `respond_to?(:process_attributes)`?

Reply:
`respond_to?` used to be much slower than calling a method that just returns `true` or `false`. However, I just benchmarked it again and it looks like YJIT makes it faster.

Reply:
Gotta love YJIT! 😁

Reply:
I still wonder if changing the caching strategy should be a separate thing in case someone wants to use `process_attributes` without using class-isolated attribute caching since that part costs ~10% and isn't necessary if you have one global, deterministic processing strategy.

Reply:
That's an interesting idea: a way to opt-in to an isolated attribute cache. Similar to how you can opt in to deferred rendering.